relayer-cli/src/commands/query/packet/commitments.rs

```diff
@@ -27,6 +27,13 @@ pub struct QueryPacketCommitmentsCmd {
 
     #[clap(required = true, help = "identifier of the channel to query")]
     channel_id: ChannelId,
+
+    #[clap(
+        short = 'l',
+        long,
+        help = "limit the number of commitments to retrieve"
+    )]
+    limit: Option<u64>,
```

Comment:
I am wondering if it's better to just have a simple check, i.e check for `has_commitments()` instead:
```suggestion

    #[clap(
        short = 'c',
        long,
        help = "check presence of commitments (packets not yet acknowledged or timed out)"
    )]
    check: bool,
```